app/controllers/application_controller/session_size.rb

```diff
@@ -1,16 +1,6 @@
 module ApplicationController::SessionSize
   extend ActiveSupport::Concern
 
-  # Session data size logging constants
-  case Rails.env
-  when "test", "development"
-    SESSION_LOG_THRESHOLD = 50.kilobytes
-    SESSION_ELEMENT_THRESHOLD = 5.kilobytes
```

Comment:
Note, we're losing these values for test and development.  I don't know if using the 100 and 10 values below is a problem for others.

Reply:
I couldn't think of a graceful solution other than to remove the `test` and `development` cases.  Theoretically during a test this could be set to lower manually if you wanted to track session size and `WARN` when necessary, but that's obviously not the best solution either since you'd have to do it per group of tests as opposed to just using `Rails.env` to automatically do it.

Reply:
@robbmanes Yeah, I don't use this so I'm not sure how much trouble it is.  It would be annoying to specify 4 settings, 2 for dev/test and 2 for production.  If we can't use 100/10 values in dev/test, maybe we can just take the `Settings.session.log_threshold/2` `Settings.session.element_threshold/2` if you're running in dev/test.  It's silly but 🤷‍♂️